### PR TITLE
Fixed some errors

### PR DESCRIPTION
**Pull request by rin0chan**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -59,6 +60,14 @@ description = "Speel live muziekvideo's overal af"
 tags = "muziekvideo's, uitzenden, live, real-time, spotify, android, TV, windows, linux, mac os, televisie"
 dateFormat = "2 January 2006"
 
+[Languages.cn]
+weight = 4
+languageCode = "zh-cn"
+title = "Vidify"
+description = "123"
```
Comment (marioortizmanero) [inline config.toml:67]: This is missing and it's fairly important. Can you edit this section before I merge?

File: data/en/apis.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ items:
       description: "Any MPRIS compatible media player for Linux or BSD (99% of them, like Spotify, Clementine, VLC...)."
       image: "/images/apis/mpris.svg"
       url: "/wiki/linux-media-players"
-
+
```
Comment (marioortizmanero) [inline data/en/apis.yml:14]: unnecessary whitespace

File: data/en/service.yml [hunk 3]
```diff
@@ -4,6 +4,7 @@ topTitle: A LIST OF SERVICES
 title: Our Service.
 subtitle: Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illum reiciendis quasi itaque, obcaecati atque sit!
 item:
+
```
Comment (marioortizmanero) [inline data/en/service.yml:7]: unnecessary change

**Review by marioortizmanero — CHANGES_REQUESTED** [review]
Left a few notes before merging. Please for the next time don't change stuff that is unrelated to the PR, like moving files into `es`, `en`... directories because it makes it hard to read the diffs afterwards. You can just do multiple PRs to make it more clean. Don't worry about it in this case though.

Edit: there actually seem to be multiple copies of the files in `content/blog`. Can you just remove the directories in there then?

File: config.toml (hunk 1 above)
Comment (totoneei) [reply config.toml:67]: OK

**Pull request closed without merging**